website/docs/r/ec2_transit_gateway.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "aws"
+page_title: "AWS: aws_ec2_transit_gateway"
+sidebar_current: "docs-aws-resource-ec2-transit-gateway-x"
```

Comment:
The website highlights in the sidebar based on `sidebar_current` as a prefix. Since we have multiple pages with `docs-aws-resource-ec2-transit-gateway` as a prefix, they would all highlight without this differentiation. (The usage of `-x` specifically is just a pattern we follow in other places).